test/integration/model/optimistic_locking.test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const Support = require(__dirname + '/../support')
+  , DataTypes = require(__dirname + '/../../../lib/data-types')
+  , chai = require('chai')
+  , expect = chai.expect;
+
+describe(Support.getTestDialectTeaser('Model'), function() {
+  describe('optimistic locking', function () {
+    var Account;
```

Comment:
Why the mixing of es5 and es6, you could make the code less verbose using arrows.